res/database/Default/user/jorgaeff.xml

```diff
@@ -339,19 +393,23 @@
 		<news id="news-jorgaeff-chorris-02" type="2" thread_id="news-jorgaeff-chorris" creator="8936" created_by="jorgaeff">
 			<title>
 				<de>Nuck Chorris zählt bis zur Unendlichkeit</de>
+				<en>Nuck Chorris counts to infinity</en>
 			</title>
 			<description>
-				<de>Laut eigenen Angaben hat Nuck Chorris bereits 2x bis zur Unendlichkeit gezählt. Die Mathe-Experten dieser Erde rätseln immer noch darüber ob das stimmt.</de>
+				<de>Laut eigenen Angaben hat Nuck Chorris bereits 2x bis zur Unendlichkeit gezählt. Die Mathe-Experten dieser Erde rätseln immer noch darüber, ob das stimmt.</de>
+				<en>According to his own statements, Nuck Chorris has already counted 2x to infinity. The math experts of this world are still puzzling over whether this is true.</en>
 			</description>
 			<data genre="1" price="0.3" quality="30" fictional="1" />
 		</news>
 
 		<news id="news-jorgaeff-drops-01" type="0" thread_id="news-jorgaeff-drops" creator="8936" created_by="jorgaeff">
 			<title>
 				<de>Jonnie Drops bei Dreharbeiten schwer verletzt</de>
+				<en>Jonnie Drops seriously injured during filming</en>
 			</title>
 			<description>
 				<de>Beim Dreh von "Mein Grog - dein Grog" hat sich Jonnie Drops an einer zerbrochenen Grogflasche verletzt. Die Dreharbeiten verzögern sich.</de>
+				<en>During the filming of "My Grog - Your Grog" Jonnie Drops injured himself due to a broken grog bottle. The shooting is delayed.</en>
```

Comment:
just wondering: here and in the folling items you use "shooting" after having it replaced elsewhere by "filming"

Reply:
Always depends on context.
In that other item with the stunt dog, the headline could really easily have been confused with a killing spree.
And here perhaps I was fully aware (I don't remember) and used the terms interchangeably on purpose to avoid repetition.